x/did/client/cli/query.go

```diff
@@ -19,7 +19,11 @@ func GetQueryCmd() *cobra.Command {
 		RunE:                       client.ValidateCmd,
 	}
 
-	cmd.AddCommand(CmdGetDid())
+	cmd.AddCommand(
+		CmdGetDidDoc(),
+		CmdGetDidDocVersion(),
+		CmdGetAllDidDocVersionsMetadata(),
+	)
```

Comment:
`DisableFlagParsing` above will try and pass persistent flags as arguments correct?

Reply:
// DisableFlagParsing disables the flag parsing.
	// If this is true all flags will be passed to the command as arguments.

Reply:
This came from examples.